src/pages/Models/Details/_model-details.scss

```diff
@@ -7,6 +7,10 @@
   margin-bottom: 2rem;
   padding-bottom: 14rem;
 
+  .p-main-table caption {
```

Comment:
I don't see a caption on the model details page.

Reply:
It's the first screenshot above. You can see it in a controller model if you've deployed your own controller. Or if you create an empty model.